packages/launchpad/src/setup-actions.ts

```diff
@@ -0,0 +1,76 @@
+import { isMobile } from '@automattic/viewport';
+import { addQueryArgs } from '@wordpress/url';
+import type { LaunchpadTracksData, Task } from './types';
+
+export const setUpActionsForTasks = (
+	tasks: Task[],
+	siteSlug: string | null,
+	tracksData: LaunchpadTracksData
+): Task[] => {
+	const { recordTracksEvent, checklistSlug, tasklistCompleted } = tracksData;
+
+	//Record click events for tasks
+	const recordTaskClickTracksEvent = ( task: Task ) => {
+		recordTracksEvent( 'calypso_launchpad_task_clicked', {
+			checklist_slug: checklistSlug,
+			checklist_completed: tasklistCompleted,
+			task_id: task.id,
+			is_completed: task.completed,
+			context: 'customer-home',
```

Comment:
I think `context` should be supplied as an argument, likely in `tracksData.launchpadContext`.
```suggestion
			context: launchpadContext,
```